### PR TITLE
added functional support plan for self hosted customers, removed VIP …

### PR DESCRIPTION
**Pull request by umairsy**

### CODE REVIEW

**Comment by umairsy** [comment]
@rmehta please review and merge

File: erpnext_com/www/pricing/support-for-self-hosted.md [hunk 1]
```diff
@@ -0,0 +1,23 @@
+# Support for Self-Hosted
```
Comment (rmehta) [inline erpnext_com/www/pricing/support-for-self-hosted.md]: sounds abrupt, "Functional Support for self-hosted ERPNext"

File: erpnext_com/www/pricing/support-for-self-hosted.md [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+
+Prefer to self host your ERPNext account? You can still avail support from us.
+
```
Comment (rmehta) [inline erpnext_com/www/pricing/support-for-self-hosted.md]: avoid the work "avail" use "get"

**Comment by umairsy** [comment]
fixed. please review.

**Pull request merged**